react_front_end/src/__tests__/services/TikTok.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @jest-environment node
+ */
+
+import { metricParser } from "../../services/TikTok";
+
+test("video metadata", async () => {
+  const url = "https://www.tiktok.com/@tiktok/video/6807491984882765062";
+  const data = await metricParser(url);
+  expect(data.id).toBe("6807491984882765062");
```

Comment:
@bernardbaker this test is failing for me right now when I run 'yarn test', "cannot read property 'id' of undefined", see below. Is this supposed to happen? 
![test](https://user-images.githubusercontent.com/13459787/79143836-df4dca00-7df8-11ea-83b4-24017538c144.PNG)

Reply:
@dzx11 Could you provide the full stack trace. From the console.

Reply:
@dzx11 Also try increasing the `jest.setTimeout(n)`.

https://jestjs.io/docs/en/jest-object#jestsettimeouttimeout

Reply:
@bernardbaker It looks like it's trying to connect to localhost:8888 for some reason. Are there some config changes that need to be made to change this? I think I saw some discussion with Brandon regarding this. Using jest.setTimeout didn't make a difference.
Full trace below.
![test2](https://user-images.githubusercontent.com/13459787/79188856-266ca700-7e5b-11ea-8cb5-43ffc5acc7ea.PNG)